client/app/services/query.js

```diff
@@ -387,7 +393,8 @@ export class Parameter {
       if (has(query, key)) {
         if (this.multiValuesOptions) {
           try {
-            this.setValue(JSON.parse(query[key]));
+            const valueFromJson = JSON.parse(query[key]);
+            this.setValue(isArray(valueFromJson) ? valueFromJson : query[key]);
```

Comment:
Turns out numbers are also parsed and converted :sweat_smile:, so this makes sure we don't convert it unless it's an array.